<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDbPriceChange</class>
 <widget class="QDialog" name="DialogDbPriceChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="tableViewData"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QComboBox" name="comboBoxViewMode">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>38</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelDescViewMode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Показывать</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUnitViewMode">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>продуктов</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxDateTimeUsage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddRow">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
